graphene_django/templates/graphene/graphiql.html

```diff
@@ -24,41 +24,13 @@
 </head>
 <body>
   <script>
+
```

Comment:
?

Reply:
This was me "boyscouting" to make the code more readable, but I decided it was unnecessary when I re-did these changes.